addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -66,100 +50,141 @@
+                                                <span t-if="move.product_packaging_id">
+                                                    (<span t-field="move.product_packaging_qty" t-options='{"widget": "integer"}'/> <span t-field="move.product_packaging_id.name"/>)
                                                 </span>
-                                                <span t-if="o.state == 'done'">
-                                                    (<span t-field="ml.product_packaging_qty" t-options='{"widget": "integer"}'/> <span t-field="ml.move_id.product_packaging_id.name"/>)
-                                                </span>
-                                            </span>
-                                        </td>
-                                        <td t-if="o.picking_type_id.code != 'incoming'" groups="stock.group_stock_multi_locations">
-                                            <span t-esc="ml.location_id.display_name">WH/Stock</span>
-                                                <t t-if="ml.package_id">
-                                                    <span t-field="ml.package_id">Package A</span>
-                                                </t>
-                                        </td>
-                                        <td t-if="o.picking_type_id.code != 'outgoing'" groups="stock.group_stock_multi_locations">
-                                            <div>
-                                                <span t-field="ml.location_dest_id">WH/Outgoing</span>
-                                                <t t-if="ml.result_package_id">
-                                                    <span t-field="ml.result_package_id">Shelf 1</span>
-                                                </t>
-                                            </div>
-                                        </td>
-                                        <td class=" text-center h6" t-if="has_serial_number">
-                                            <span t-if="has_serial_number and (ml.lot_id or ml.lot_name)" t-esc="ml.lot_id.name or ml.lot_name" t-options="{'widget': 'barcode', 'humanreadable': 1, 'width': 400, 'height': 100, 'img_style': 'width:100%;height:35px;'}">
-                                                <div class="bg-light border-1 rounded d-flex flex-column align-items-center justify-content-center px-1 py-2 opacity-75 text-muted text-center">
-                                                    (serial barcode)
+                                            </td>
+                                            <td t-if="from_col_exists" groups="stock.group_stock_multi_locations">
+                                                <div t-if="not move_has_serial_number">
+                                                    <span t-field="move.location_id.display_name">WH/Stock</span>
```

Comment:
This field gets only rendered when we have a single non-tracked move line because in this case we're not doing move lines breakdown and there is only one line (this is what `move_has_serial_number` is for). Perhaps the naming is bad so I'll change it to `move_has_tracked_lines`.